### ISSUE TITLE
Problem with left zoom position

### ISSUE THREAD

**Issue by dmiszkur** [0401morita/imagezoom-plugin#1]
Hi,

In line 46 i think you should change your code for:
 zoomViewPositionX = (offset.left - cursorSize[0]*settings.magnification - settings.zoomviewmargin);
In previous version i have problem with position on the left.

**Comment by samparmenter** [comment]
Yeah, the issue is coming from the x position of the zoom window being "master image left offset" - "master image width" - "margin between master image and zoom window".

It should be "master image left offset" - "zoom window width" - "margin between master image and zoom window".

Otherwise the zoom window ends up too far to the left.
